lib/core/src/lib/datatable/components/datatable/datatable.component.ts

```diff
@@ -1010,6 +1022,20 @@ export class DataTableComponent implements OnInit, AfterContentInit, OnChanges,
         return !drop.getSortedItems()[index].disabled;
     }
 
+    onDragDrop(droppedEvent: CdkDragDrop<any>): void {
```

Comment:
Can you add a unit test for the drag functionality?

Reply:
Done